courses/models.py

```diff
@@ -81,8 +81,12 @@ def get_course_language(course_id):
 
     @property
     def first_university(self):
```

Comment:
Ok for the comment on using property. I rather leave this as it is - I don't want to alter the interface in this PR.